.github/workflows/main.yml

```diff
@@ -0,0 +1,75 @@
+name: Bedrock Node.js CI
+
+on: [push]
+
+jobs:
+  test-node:
+    runs-on: ubuntu-latest
+    services:
+      mongodb:
+        image: mongo:4.2
+        ports:
+          - 27017:27017
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
```

Comment:
Out of curiosity, do we actually need to support Node 10?

Reply:
In a bedrock module I say we can drop it, but this does not necessarily apply to general utility modules.

Reply:
I don't see the harm in testing Maintenance LTS releases.  If we start actively using newer unsupported features then drop older versions as needed.